doc/faq.rst

```diff
@@ -44,3 +44,17 @@ two options:
        started, but that no security precautions has been taken to
        e.g. prevent serving other files out into the local
        network. Take appropriate actions if this is a concern.
+
+How can I save, then restore the previous playing Sonos state ?
+---------------------------------------------------------------
+
+This is useful for scenarios such as when you want to switch to radio,
+an announcement or doorbell sound and then back to what was playing previously.
+Documentation of the Snapshot module `~soco.snapshot.Snapshot`
+
+SoCo provides a snapshot module that captures the current state of a player and
+then when requested re-instates that state. Examples of it's use are:
+ - `basic snap example  <https://github.com/SoCo/SoCo/blob/master/examples/snapshot/basic_snap.py>`_
```

Comment:
I think the content looks good.

There is a minor syntax error. You should make a blank line after the colon and don't indent the list and I think use * as lists, so it becomes:

```rst
SoCo provides a snapshot module that captures the current state of a player and
then when requested re-instates that state. Examples of it's use are:

* `basic snap example  <https://github.com/SoCo/SoCo/blob/master/examples/snapshot/basic_snap.py>`_
* `multi zone example  <https://github.com/SoCo/SoCo/blob/master/examples/snapshot/multi_zone_snap.py>`_
```

If you have sphinx and napoleon installed you can test the compilation by doing "make html" in the doc directory.